packages/viewer/src/modules/drawing/DrawingModule.vue

```diff
@@ -107,94 +158,95 @@ watch(hasLoaded, () => {
         addKmlToDrawing()
     }
 })
-// watching feature IDs so that we can react whenever one is removed through the "trash button"
-watch(featureIds, (next, last) => {
+
+// Watching feature IDs so we can react whenever one is removed through the "trash button"
+watch(featureIds, (next: string[], last: string[]) => {
     const removed = last.filter((id) => !next.includes(id))
     if (removed.length > 0) {
         log.debug(`${removed.length} feature(s) have been removed, removing them from source`)
         const source = drawingLayer.getSource()
+        if (!source) return
```

Comment:
Please add empty lines or the curly braces around this line, otherwise it get's visually lost